assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -74,80 +109,239 @@ const GridItemHalfRow = styled(GridItem)`
-            <FlyInRightAnimation>
-              {__('Fly in', 'web-stories')}
-            </FlyInRightAnimation>
-          </ContentWrapper>
+        <GridItem
+          aria-label={__('Fly In from Right Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.FLY_IN.value,
+              direction: DIRECTION.RIGHT_TO_LEFT,
+            })
+          }
+        >
+          <ContentWrapper>{__('Fly in', 'web-stories')}</ContentWrapper>
+          <FlyInRightAnimation>
+            {__('Fly in', 'web-stories')}
+          </FlyInRightAnimation>
         </GridItem>
-        <GridItemFullRow>
-          <ContentWrapper>
-            <PulseAnimation>{__('Pulse', 'web-stories')}</PulseAnimation>
-          </ContentWrapper>
+        <GridItemFullRow
+          aria-label={__('Pulse Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.PULSE.value,
+            })
+          }
+        >
+          <ContentWrapper>{__('Pulse', 'web-stories')}</ContentWrapper>
+          <PulseAnimation>{__('Pulse', 'web-stories')}</PulseAnimation>
         </GridItemFullRow>
-        <GridItemHalfRow>
-          <ContentWrapper>
-            <RotateInLeftAnimation>
-              {__('Rotate', 'web-stories')}
-            </RotateInLeftAnimation>
-          </ContentWrapper>
+        <GridItemHalfRow
+          aria-label={__('Rotate In Counter Clockwise Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.ROTATE_IN.value,
+              rotation: ROTATION.COUNTER_CLOCKWISE,
+            })
+          }
+        >
+          <ContentWrapper>{__('Rotate', 'web-stories')}</ContentWrapper>
+          <RotateInLeftAnimation>
+            {__('Rotate', 'web-stories')}
+          </RotateInLeftAnimation>
         </GridItemHalfRow>
-        <GridItemHalfRow>
+        <GridItemHalfRow
+          aria-label={__('Rotate In Clockwise Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.ROTATE_IN.value,
+              rotation: ROTATION.CLOCKWISE,
+            })
+          }
+        >
+          <ContentWrapper>{__('Rotate', 'web-stories')}</ContentWrapper>
           <RotateInRightAnimation>
-            <ContentWrapper>{__('Rotate', 'web-stories')}</ContentWrapper>
+            {__('Rotate', 'web-stories')}
           </RotateInRightAnimation>
         </GridItemHalfRow>
+        <GridItemFullRow
+          aria-label={__('Twirl In Effect', 'web-stories')}
+          onClick={() =>
+            onAnimationSelected({
+              animation: ANIMATION_EFFECTS.TWIRL_IN.value,
+            })
+          }
+        >
+          <ContentWrapper>{__('Twirl In', 'web-stories')}</ContentWrapper>
+          <TwirlInAnimation>{__('Twirl In', 'web-stories')}</TwirlInAnimation>
```

Comment:
I just realized we may be able to use `ANIMATION_EFFECTS.TWIRL_IN.name` here for the text contents of each animation effect option